tests/wpt/mozilla/tests/mozilla/anchor_target.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Anchor target attribute setter/getter</title>
```

Comment:
There does not appear to be a reason to make this test Servo-specific. I'd suggest you submit it upstream, but I think this is already tested by `html/dom/reflection-text.html`.